### PR TITLE
Add screenshots with ~1.5 lines long desc to expanded location view

### PR DESCRIPTION
**Pull request by julioromano**
This will help in catching alignment regressions.

Related to:
- https://github.com/vector-im/element-meta/issues/1678

### CODE REVIEW

**Comment by csmith** [comment]
Good catch, but I'm not sure justifying the text is correct here. We want it centered, and if you have a description that wraps one or two words onto the second line it looks pretty bad:

image

I wonder why it's not centered properly - something to do with the overflow ellipses? :thinking:

File: features/location/impl/src/main/kotlin/io/element/android/features/location/impl/show/ShowLocationView.kt [hunk 1]
```diff
@@ -138,12 +137,12 @@ fun ShowLocationView(
             state.description?.let {
                 Text(
                     text = it,
-                    textAlign = TextAlign.Center,
+                    textAlign = TextAlign.Justify,
```
Comment (jmartinesp) [inline features/location/impl/src/main/kotlin/io/element/android/features/location/impl/show/ShowLocationView.kt:140]: I'd just use `TextAlign.Start` here, justify looks a bit weird, as @csmith mentioned.

Comment (julioromano) [reply features/location/impl/src/main/kotlin/io/element/android/features/location/impl/show/ShowLocationView.kt:140]: TextAlign.Start doesn't change things much:
Screenshot 2023-07-20 at 09 33 46
Screenshot 2023-07-20 at 09 33 41
Screenshot 2023-07-20 at 09 33 38

Comment (jmartinesp) [reply features/location/impl/src/main/kotlin/io/element/android/features/location/impl/show/ShowLocationView.kt:140]: Oh sorry, I misunderstood it, you *always* want it centered, even if the text has several lines and overflows. It seems a bit weird to me, but in that case we should go back to center aligning the text and trying to figure out why we have that extra indentation instead.

**Comment by julioromano** [comment]
Ok I double checked with Callum so I left it as is, but I still added one screenshot with the second line half filled so that we can catch regressions on this.

**Review by csmith — APPROVED** [review]

**Pull request merged**